### ISSUE TITLE
Plan 2019-10 release

### ISSUE THREAD

**Issue by cgolubi1** [buttonmen-dev/buttonmen#2472]
Do all testing and deployments for the 2019-10 buttonweavers.com prod release.

**Comment by cgolubi1** [comment]
Hmm, something ain't right.  http://staging.buttonweavers.com/ui/buttons.html?button=Julia gives me:

"Internal error when calling loadButtonData"

And i'm very confused about it, because there's nothing in the apache error log on the staging server.

Here's the access log entries from the call:

```
"GET /ui/buttons.html?button=Julia HTTP/1.1" 200 662 "http://staging.buttonweavers.com/ui/buttons.html?set=Peloton" 
"GET /ui/js/extern/jquery-ui.css HTTP/1.1" 200 6075 "http://staging.buttonweavers.com/ui/buttons.html?button=Julia" 
"GET /ui/gui.1566429269.css HTTP/1.1" 200 4103 "http://staging.buttonweavers.com/ui/buttons.html?button=Julia" 
"GET /ui/js/extern/jquery-1.9.0.min.js HTTP/1.1" 200 32938 "http://staging.buttonweavers.com/ui/buttons.html?button=Julia" 
"GET /apple-touch-icon.png HTTP/1.1" 200 13564 "-" 
"GET /favicon.ico HTTP/1.1" 200 1150 "-" 
"GET /ui/js/Loader.1566429269.js HTTP/1.1" 200 674 "http://staging.buttonweavers.com/ui/buttons.html?button=Julia" 
"GET /ui/js/Login.1566429269.js HTTP/1.1" 200 4451 "http://staging.buttonweavers.com/ui/buttons.html?button=Julia" 
"POST /api/responder HTTP/1.1" 200 63 "http://staging.buttonweavers.com/ui/buttons.html?button=Julia" 
"POST /api/responder HTTP/1.1" 200 103 "http://staging.buttonweavers.com/ui/buttons.html?button=Julia" 
"POST /api/responder HTTP/1.1" 200 - "http://staging.buttonweavers.com/ui/buttons.html?button=Julia" 
```

That last POST, i'm sending:

```
{"type":"loadButtonData","buttonName":"Julia","automatedApiCall":false}	
```

And the server is returning no response.  Which explains the situation, but... what?

**Comment by cgolubi1** [comment]
Figured it out; the flavor text contains non-ascii characters.  Do we seriously not have a unit test for that?

**Comment by blackshadowshade** [comment]
After brief discussion with @cgolubi1, I'll resolve this for now by replacing the ellipsis with three dots. We'll think about how to support Unicode later.

**Comment by cgolubi1** [comment]
Okay, i've finally updated the staging site.  Before we can go to production, i need to unhose backups in a more systematic way that will keep them unhosed for more than a week at a time; i'll try to get to that this week.  Meanwhile, let me know if anyone else sees any issues on the staging site.

**Comment by dwvanstone** [comment]
I can read all the button text and see all the graphics of the new Peloton set. The help page looks good. Yay!

**Comment by blackshadowshade** [comment]
Looks good to me, played a turn, joined a Peloton game, Julia's flavour text looks good, help pages are accessible.

**Comment by cgolubi1** [comment]
This turned into the October release while i was finding time to fix the backups, but i did do that, in #2479 so we should be good to go.

**Comment by blackshadowshade** [comment]
This looks like it's done, so I'll close it.
